### ISSUE TITLE
unknown imagen type None

### ISSUE THREAD

**Issue by night3759** [lucidrains/imagen-pytorch#220]
when I used the function load_imagen_from_checkpoint, a valueerror occured. How to fix it?
![image](https://user-images.githubusercontent.com/46178719/190935206-6693fa7c-f598-4104-8211-7b575b181b9c.png)

**Comment by TheFusion21** [comment]
Did you use the ImagenConfig or ElucidatedImagenConfig to instantiate Imagen?

**Comment by night3759** [comment]
> Did you use the ImagenConfig or ElucidatedImagenConfig to instantiate Imagen?

No, actually I just used the Imagen Function in the usage. I seem to know how to fix it. Thanks!

**Comment by cxchhh** [comment]
I also have this problem, how should I do to "instantiate Imagen" in CLI?
